components/library/libuv/Makefile

```diff
@@ -12,20 +12,20 @@
 # Copyright 2020, Michal Nowak
 #
 
-BUILD_BITS=		64
+BUILD_BITS=		32_and_64
 
```

Comment:
Why did you change this? The only actual consumer seems to be network/dns/bind which is 64-bit only.
Do you want to use libuv in 32-bit?
Furthermore, if you make changes like this, you'll also need to update manifest/sample-manifest.p5m by running "gmake sample-manifest" and check for its changes (compared to the old version). Then you either need to update the package's manifest (libuv.p5m in this case). Without the last step the addition of 32-bit files won't make it into the package.